### DISCUSSION TITLE
[Resolved] Currently preparing for a release v1.3.0

### DISCUSSION THREAD

**Discussion by fre171csiro** [gbrrestoration/mds-is-issues#8] [Announcements]
[Resolved]
The system is currently undergoing release changes which should not take long to complete. During this time you may experience issues (500 Internal Server Error ).  We will notify you again once the release is completed and you should be able to re-run your actions

Cheers
RRAP IS devops team

**Comment by PeterBaker0** [comment]
The system should be operational again with the exception of the search service, which will be available in a few minutes.

**Reply by PeterBaker0** [reply]
The system should be fully operational again. Let us know if you experience any issues.
